app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,96 @@
+@import 'font-awesome';
+
+* {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please you need to work on your project's design(`Register`, `Home` and `Detail` pages), I compare your and [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) designs pictures below🙏🏼


| Your Design  |
| ------------- | 

#
![image](https://user-images.githubusercontent.com/63915024/221377839-5488bf26-9d69-4aac-88df-6fc8bbe2fa8d.png)
#

| [Behance Design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)  |
| ------------- |  

#
![image](https://user-images.githubusercontent.com/63915024/221377829-38a0a6ba-78a8-42b7-ab01-da59614176c2.png)
#

Reply:
@mukhammadrizooff ,
I have implemented what I could on the project because I was limited on time.